src/app/api-explorer-jsviewer.ts

```diff
@@ -38,9 +38,9 @@ export function initializeJsonViewer() {
 
 export function initializeResponseHeadersViewer() {
     const jsonViewer = getAceEditorFromElId('response-header-viewer');
-    const ariaLabel = getTextAreaAriaLabel('Response headers viewer');
+    const name = getTextAreaAriaLabel('Response headers viewer');
```

Comment:
response-header-viewer is not a POSH element.

![image](https://user-images.githubusercontent.com/8527305/47449812-71cb5b00-d778-11e8-8df0-11ab5ec56d92.png)

Reply:
I think this would apply to the Response body as well.

Reply:
@MIchaelMainer  I also thought the same and was surprised to find otherwise.

This is how we are setting the name attribute

```
editor.textInput.getElement().setAttribute('name', name);
```

Finding out what element we are setting the attribute on was fairly trivial. I checked the value of `editor.textInput.getElement()`. Which is the value below.

```
<textarea class="ace_text-input" wrap="off" autocorrect="off" autocapitalize="off" spellcheck="false" name="The response headers returned by Microsoft Graph" style="opacity: 0;"></textarea>
```
and not the div in the screenshot.

This means the `setAttribute` method is called on `<textarea>`